portfolio/src/main/webapp/gallery-script.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * Load project details from local registry of projects (projects.json)
+ */
+const loadProjects = async() => {
+  const request = new XMLHttpRequest();
+  request.open("GET", "projects.json", true);
```

Comment:
Why do you need to make HTTP request to access local file? Can't you directly access it?

Reply:
We're only using client-side code for the portfoli,o at least at the moment, so this seemed like the easiest way to get the file onto the client's machine. I didn't know of any other way to include the file without making a second request once the javascript loads.